pimd/pim_mroute.c

```diff
@@ -637,8 +638,10 @@ static int pim_mroute_msg(struct pim_instance *pim, const char *buf,
 		if (igmp)
 			pim_igmp_packet(igmp, (char *)buf, buf_size);
 		else if (PIM_DEBUG_IGMP_PACKETS) {
-			zlog_debug("No IGMP socket on interface: %s with connected source: %pFX",
-				   ifp->name, connected_src);
+			char ifaddr_str[INET_ADDRSTRLEN];
+			pim_inet4_dump("<ifaddr?>", ifaddr, ifaddr_str, sizeof(ifaddr_str));
+			zlog_debug("No IGMP socket on interface: %s with connected source: %s",
```

Comment:
please use %pI4 instead of pim_inet4_dump and %s

Reply:
Fixed already. CI failed but seems not related.